src/index.css

```diff
@@ -8,5 +8,51 @@
   font-family: 'Work Sans', sans-serif;
 }
 
+/* Global Styles */ 
 
+/* Ensures a parent element with this class name stretches to the height of floated children */
+.clearfix:after {
+  content: "";
+  display: block;
+  clear: both;
+}
+
+.container {
+  max-width: 1280px;
+  margin: 0 auto;
+}
```

Comment:
Having a standard max width on the site is useful and expected (so that on really large screens it doesn't just go on forever). 1280px is a width I like, but play with what works for you and for the content.